### PR TITLE
docs: Update `README`

### LINKED ISSUE #470: [README] Installation howto is outdated and update howto doesn't exist
### The problem

Ref: https://github.com/derekantrican/GAS-ICS-Sync/discussions/469
* The «[If you want to use this, please copy the script from here](https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true)» in README leads to incorrect (outdated) version of GAS
* The [Wiki: Setting up the script manually](https://github.com/derekantrican/GAS-ICS-Sync/wiki/Setting-up-the-script-manually) is outdated as is missing [filters.gs](https://github.com/derekantrican/GAS-ICS-Sync/blob/master/filters.gs)
* No update guidance is provided. The process is intuitively clear (copy over files content from repo to GAS project), whilst might be non-obvious for beginners (and tripped me over by a missing file from the item above)

### PR DESCRIPTION
**Pull request by yermulnik**
- Update link to template GAS project
- Add GAS code update instructions

I obviously have insufficient permissions to PR to Wiki, hence here's a relevant patch: 
[0001-docs-Add-filters.gs-to-Wiki.patch.txt](https://github.com/user-attachments/files/17904818/0001-docs-Add-filters.gs-to-Wiki.patch.txt)

Fixes https://github.com/derekantrican/GAS-ICS-Sync/issues/470

Refs:
- https://github.com/derekantrican/GAS-ICS-Sync/discussions/469

### CODE REVIEW

**Comment by derekantrican** [comment]
1. That link is not publicly accessible
2. Even if it was, I doubt that link points to a file that we (the developers) control to update

**Comment by yermulnik** [comment]
> 1. That link is not publicly accessible
> 2. Even if it was, I doubt that link points to a file that we (the developers) control to update

That's confusing 😕
That link was provided by this repo collaborator @jonas0b1011001 via https://github.com/derekantrican/GAS-ICS-Sync/discussions/469#discussioncomment-11355700
Once I created an issue to get README/Wiki updated, @jonas0b1011001 also asked me to create PR via https://github.com/derekantrican/GAS-ICS-Sync/issues/470#issuecomment-2498159923

Any misalignment here? Now I feel like I was doing something wrong and wasted time for nothing as of the incorrect(?) pointer from this repo dev 😞

**Comment by yermulnik** [comment]
I could've reverted the link change (still the link in current README points to an **outdated GAS project**) since the other part of PR (and the attached patch) does add value I guess. Ain't it?

**Comment by derekantrican** [comment]
@jonas0b1011001 - let me know what your input is here. If this doc link is owned by you (and you want to move the ownership that way so I don't have to be involved to update it) then I'm fine with that. But this link needs to be publicly accessible.

@yermulnik Yes, the link intentionally points to an outdated GAS project. You can think of that link as "the latest release" and the code in our repo as "the development branch". We could track this discrepancy through different branches on GitHub (eg "release" and "dev" or whatever) but we haven't done that. We update https://github.com/derekantrican/GAS-ICS-Sync/releases whenever we update that script file (and you can see that was last done around a year ago)

**Comment by yermulnik** [comment]
@derekantrican Got it, thanks for explanation. Maybe it would make sense to update README/Wiki with details on release vs development versions as e.g. for me it appeared to be a bit of a fuss to figure out what was going wrong as I wanted to use most recent version and simply missed that `filters.gs` file was missing from my GAS project. Thanks.

**Comment by jonas0b1011001** [comment]
Sorry for the late response.

> @jonas0b1011001 - let me know what your input is here. If this doc link is owned by you (and you want to move the ownership that way so I don't have to be involved to update it) then I'm fine with that. But this link needs to be publicly accessible.

That link indeed is pointing to a script on my google drive and is used by me to allow people in the issues/discussion section to quickly test if the new build is solving their problems. As i was trying to express [here](https://github.com/derekantrican/GAS-ICS-Sync/discussions/469#discussioncomment-11355700), the link is not intended to replace the 'stable' link in the readme.

> for me it appeared to be a bit of a fuss to figure out what was going wrong as I wanted to use most recent version and simply missed that `filters.gs`

I was not aware that the only missing piece in the wiki was `filters.gs` missing from the list of files to copy. I thought you had more general problems with the instructions which is why i asked you for help to make it more clear for someone who is new to the project.

---
bottom line:
1. We should probably release a new version soon, there have been some major updates since 5.8
2. [Settings v2 branch](https://github.com/derekantrican/GAS-ICS-Sync/blob/settings-v2/About.gs) has update instructions as part of the script files. Once we merge that in we should probably delete the wiki page to avoid confusion or outdated instructions.
3. As for the additional link, we could add a link to the latest 'beta'/'dev' build to reduce the need of manual updating alltogether.

Thoughts?
